craft_cli/errors.py

```diff
@@ -96,6 +96,37 @@ def __eq__(self, other: object) -> bool:
         return NotImplemented
 
 
+class CraftCommandError(CraftError):
+    """A CraftError with precise error output from a command.
+
+    This exception class augments CraftError with the addition of a ``stderr``
+    parameter. This parameter is meant to hold the standard error contents of
+    the failed command. As such, it sits between "brief" and "details" from the
```

Comment:
`brief` and `details` seem orthogonal - do you mean `brief` and `verbose` or `message` and `details`?

Reply:
I do mean `message` and `details`, will update. Thankss